packages/notifications/push/amplify_push_notifications_pinpoint/lib/src/pinpoint_provider.dart

```diff
@@ -35,33 +42,39 @@ class PinpointProvider implements ServiceProviderClient {
   Future<void> init({
     required NotificationsPinpointPluginConfig config,
     required AmplifyAuthProviderRepository authProviderRepo,
-    // @visibleForTesting FlutterAnalyticsClient? mockAnalyticsClient,
+    @visibleForTesting AnalyticsClient? mockAnalyticsClient,
```

Comment:
You can just name it analyticsClient.  The visibleForTesting param already indicates it's just for testing purposes.